.eslintrc.yml

```diff
@@ -71,12 +71,16 @@ rules:
   flowtype/use-flow-type: error
   flowtype/valid-syntax: off
 
+  ##################################################
+  # ESLint builtin rules list based on `v5.6.0`
+  ##################################################
```

Comment:
@mjmahone Idea here is to add new rules every time we update ESLint in `package.json`.

Reply:
That makes sense to me. Thanks for adding the comment.